cognite/powerops/data_classes/shop_file_config.py

```diff
@@ -21,12 +22,17 @@ class ShopFileConfig(BaseModel):
         "extra_data",
         "water_value_cut_file_reservoir_mapping",
         "water_value_cut_file",
+        "modulserie",
     ]
     md5_hash: Optional[str] = None
 
     @property
     def external_id(self) -> str:
-        return f"SHOP_{self.watercourse_name}_{self.cogshop_file_type}"
+        return (
+            f"SHOP_{self.watercourse_name}_{self.extra_name}_{self.cogshop_file_type}"
+            if self.extra_name
+            else f"SHOP_{self.watercourse_name}_{self.cogshop_file_type}"
+        )
```

Comment:
Suggest the following
```suggestion
        return (
            f"SHOP_{self.watercourse_name}_{self.path.stem}_{self.cogshop_file_type}"
        )
```
Wouldn't this help the traceability of the shop file